manager_test.go

```diff
@@ -258,6 +265,21 @@ func singleNotChangedStack(t *testing.T) repository {
 	}
 }
 
+func singleNotChangedStackWithEmptyModuleSrc(t *testing.T) repository {
```

Comment:
Ah sorry the suggestion broke the code, you need a proper full replace on the whole code